### PR TITLE
Docsp 15578

### PR DESCRIPTION
**Pull request by terakilobyte**
## Pull Request Info

This simplifies navigation to the new java driver.

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-15578

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-15578/java-drivers/

### CODE REVIEW

File: source/java-drivers.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ Java MongoDB Drivers
 .. toctree::
    :titlesonly:
 
-   /java.txt
+   Java Driver <https://docs.mongodb.com/drivers/java/sync/current/>
```
Comment (ccho-mongodb) [inline source/java-drivers.txt:12]: Does 
```
:driver:`Java Driver </java/sync/current>`
```
 work in the toctree?

**Review by ccho-mongodb — APPROVED** [review]
LGTM, one question.
Also, I think you meant to link the following JIRA ticket: https://jira.mongodb.org/browse/DOCSP-17578

File: source/java-drivers.txt (hunk 1 above)
Comment (terakilobyte) [reply source/java-drivers.txt:12]: It doesn't appear to in testing :/

Comment (ccho-mongodb) [reply source/java-drivers.txt:12]: Ah, I guess it was "won't do": https://jira.mongodb.org/browse/DOP-815

**Pull request merged**